{{cookiecutter.repo_name}}/readthedocs.yml

```diff
@@ -0,0 +1,15 @@
+# readthedocs.yml
+
+version: 2
+
+build:
+  image: latest
+
+python:
```

Comment:
We normally build like so:
```
python:
    version: 3
    setup_py_install: true
```

Reply:
I believe this syntax is for V1 of RTD ymls - https://docs.readthedocs.io/en/stable/config-file/v2.html#new-settings

Reply:
Ah, interesting. I wonder how that works since the `requirements.yml ` is the one who determines the Python version.